corehq/apps/app_manager/templates/app_manager/app_view.html

```diff
@@ -251,11 +251,30 @@
             </div>
 
         {% endif %}
-        <div class="tab-pane{% if copy_app_form.is_bound %} active{% endif %}" id="copy">
-            <div>
-                <form class="form form-horizontal" method="post" action="{% url "copy_app" domain %}">
-                    {% crispy copy_app_form %}
-                </form>
+            <div class="tab-pane{% if copy_app_form.is_bound %} active{% endif %}" id="copy">
+                <div>
+                    <form class="form form-horizontal" method="post" action="{% url "copy_app" domain %}">
+                        {% crispy copy_app_form %}
+                    </form>
+                    <div style="{% if not export_apps_toggle %}display: none;{% endif %}">
```

Comment:
Rather just exclude this div entirely if `export_apps_toggle`. Also you can do this in the template:
```
{% if request|toggle_enabled:"EXPORT_ZIPPED_APPS" %}
```